### DISCUSSION TITLE
Support for .CLIP file type

### DISCUSSION THREAD

**Discussion by squashsquash** [ccseer/Seer#412] [General]
Clip Studio Paint is a very popular art program - it'd be great to get Seer support for previewing .clip files the same way it can preview .psd files. I contacted CSP support to ask if they could help from their end but only got a noncommittal answer that they'd look into it.

**Comment by Paddoki** [comment]
I would love to see support for .Clip files! I recently switched to using clip files instead of PSD so I could make use of the time-lapse feature they have built in so I would love to be able to quickly preview my clip files through Seer again

**Comment by squashsquash** [comment]
Back again about a year later, just wanted to bump this to see if it'd be possible to get seer support for .clip files

**Comment by ccseer** [comment]
the .CLIP file format is a proprietary format. It's better to reach out directly to the developers or official support channels of the software that creates .CLIP files. They would be best positioned to provide guidance or potentially offer a solution for viewing these files.

**Reply by squashsquash** [reply]
I have tried reaching out directly to the CSP team before without success. However I have discovered that Windows 11's PowerToys image previewer can successfully preview .CLIP files. Just flagging it as a potential resource - maybe there is some code in there that can help Seer preview these files as well.

**Comment by ccseer** [comment]
I haven't looked into it, but I think they use the system preview handler
from Explorer. This feature will be added in the next release 4.0.0.

**Comment by ccseer** [comment]
https://github.com/ccseer/Seer/discussions/452
